letsencrypt/DOCS.md

```diff
@@ -461,6 +461,22 @@ dns-gandi
 dns-transip
 ```
 
+## Auto update with an automation
```

Comment:
We should make this a Blueprint instead...
That makes is importable with a single button.